rosdep/base.yaml

```diff
@@ -5535,6 +5535,11 @@ qt4-dev-tools:
   gentoo: ['dev-qt/assistant:4', 'dev-qt/linguist:4', 'dev-qt/pixeltool:4']
   opensuse: [libqt4-devel-doc]
   ubuntu: [qt4-dev-tools]
+qt4-linguist-tools:
+  debian: [qt4-linguist-tools]
+  fedora: [qt]
+  gentoo: ['dev-qt/linguist-tools']
+  ubuntu: [qt4-linguist-tools]
```

Comment:
Looking at https://packages.ubuntu.com/bionic/qt4-linguist-tools , I'm guessing that as of Focal there is no more support for this package.  Thus, I'll suggest that we change this to:

```
  ubuntu:
    '*': null
    xenial: qt4-linguist-tools
    bionic: qt4-linguist-tools
```